src/Storefront/Resources/views/storefront/element/cms-element-category-navigation.html.twig

```diff
@@ -1,21 +1,16 @@
-{# @deprecated tag:v6.7.0 - Variable `active` will be removed, as it will be unused #}
```

Comment:
I already took care of this file here https://github.com/shopware/shopware/pull/6575

Reply:
i reverted the changes from my git history, so we will not run into conflicts 👍